### PR TITLE
Store log files in the directory '/mkw-sp/logs'

### PR DESCRIPTION
**Pull request by StarMKWii**

### CODE REVIEW

**Comment by stblr** [comment]
Is the file name change going to be its own PR?

File: payload/sp/storage/LogFile.cc [hunk 1]
```diff
@@ -19,7 +19,8 @@ static OSThread thread;
 static u8 stack[4096];
 
 static void *Run(void *UNUSED(arg)) {
-    auto file = Storage::Open(L"/mkw-sp/log.txt", "w");
+    Storage::CreateDir(L"/mkw-sp/logs", false);
```
Comment (stblr) [inline payload/sp/storage/LogFile.cc:22]: Nitpick: it doesn't matter since we aren't checking the return value, but the second arg should be `true`.

**Comment by StarMKWii** [comment]
> Is the file name change going to be its own PR?

That is correct.

File: payload/sp/storage/LogFile.cc (hunk 1 above)
Comment (StarMKWii) [reply payload/sp/storage/LogFile.cc]: Addressed in commit [babbee4](https://github.com/stblr/mkw-sp/pull/390/commits/babbee4e4c0d0e66071181af36c3d7ecb6f24af2).

**Review by stblr — APPROVED** [review]

**Pull request merged**